lib/ansible/modules/storage/netapp/na_ontap_gather_facts.py

```diff
@@ -46,6 +46,13 @@
             - use "help" to get a list of supported facts for your system.
         default: "all"
         version_added: 2.8
+    max_records:
+        description:
+            - Limits maximum amount of records returned per query.
+                It must be between 1 and 4294967295.
+        default: "1024"
+        type: str
```

Comment:
Why not int ?

I’d think it’s more natural to use int for a number and to convert at the last minute to match ZAPI expectations.   This is even more useful as we will transition to the new REST APIs as they are being supported in ONTAP 9.6 and newer releases.

Reply:
ok, i'll look into this

Reply:
I agree and have changed it to int ;)